tools/ctl/rundex/rundex.go

```diff
@@ -218,9 +218,15 @@ type FetchRunsOpts struct {
 	BenchmarkHash string
 }
 
+type RundexUpdate struct {
+	Rebuild *Rebuild
+	Run     *Run
+}
+
 type Reader interface {
 	FetchRuns(context.Context, FetchRunsOpts) ([]Run, error)
 	FetchRebuilds(context.Context, *FetchRebuildRequest) ([]Rebuild, error)
+	Notify() <-chan RundexUpdate
```

Comment:
I don't think this is the right place to put this abstraction. I'd recommend either

1) Adding a notify channel to Opts
2) converting the slice returns to iter returns and have the caller implement a notify channel